src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsUnaryCallLogger.php

```diff
@@ -27,13 +28,14 @@
 final class GoogleAdsUnaryCallLogger
 {
     use ArrayTrait;
-    use GoogleAdsLoggerTrait;
 
     private $logger;
     private $logLevel;
```

Comment:
Could you improve the name of this variable as well, e.g., filterLevel?

Reply:
Done.